pkg/config/config_template.yaml

```diff
@@ -1197,6 +1197,12 @@ api_key:
   #
   # connection_limit: 2000
 
+  ## @param features - list of strings - optional
+  ## @env DD_APM_FEATURES - comma separated list of strings - optional
+  ## Configure additional Trace Agent features not otherwise specified
```

Comment:
We probably want to specify here that these features are not stable and not guaranteed to persist across versions. Features that have feature flags may eventually be promoted to actual configurable features of the agent, rather than being "beta".